internal/messaging/subscriber.go

```diff
@@ -63,7 +63,7 @@ func (s *NatsSubscriber) Run(ctx context.Context) error {
 					"error", err,
 				)
 
-				if err := msg.Nak(); err != nil {
+				if err := msg.Ack(); err != nil {
 					s.logger.ErrorContext(ctx, "Failed to nak message",
```

Comment:
The error message has not been updated (it still references the Nak).

Can you also write a comment explaining why we are returning a Ack?

Reply:
The block is superfluous. I removed it as we are lacking after the error handling. 
NOTE: We will implement proper error handling in #315